dp/generate_climos.py

```diff
@@ -383,44 +385,52 @@ def convert_longitude_range(climo_data):
     return climo_data
 
 
-def convert_pr_var_units(input_file, climo_data):
-    """If the file contains a 'pr' variable, and if its units are per second, convert its units to per day.
-
+def convert_flux_var_units(input_file, climo_data):
+    """If the file contains a 'pr' or 'prsn' variable, and if its units are per
+       second, convert its units to per day.
     """
-    pr_attributes = {}  # will contain updates, if any, to pr variable attributes
+    attributes = {}  # will contain updates, if any, to pr variable attributes
+
+    vars = ['pr', 'prsn']
+    varname = ''
+    variable = None
+
+    for var in vars:
+        if var in input_file.dependent_varnames():
+            varname = var
+            variable = input_file.variables[var]
```

Comment:
This does not handle the case where a file contains _both_ a `pr` variable and a `prsn` variable.

Reply:
Oh ok! I didn't know there would be a case where that was possible.  I'll make the changes!

Reply:
It seems unlikely given the way prsn is generated with a separate script right now, to be sure. We do sometimes process files with multiple variables in them (hydrology's models often operate on several variables at once, and hydrology could make prsn data someday), so it's best not to taunt the fates.